admin-dev/themes/new-theme/scss/components/_tinymce.scss

```diff
@@ -1,1215 +1,1517 @@
 .mce-container {
-    margin: 0;
-    padding: 0;
-    border: 0;
-    outline: 0;
-    vertical-align: top;
-    background: transparent;
-    text-decoration: none;
-    color: #333;
-    font-size: 12px;
-    float: none;
+  position: static;
+  box-sizing: content-box;
+  float: none;
+  width: auto;
+  height: auto;
+  padding: 0;
+  margin: 0;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: normal;
+  color: #333;
+  text-align: left;
+  text-decoration: none;
+  white-space: nowrap;
+  vertical-align: top;
+  direction: ltr;
+  cursor: inherit;
+  background: transparent;
+  border: 0;
+  outline: 0;
+  -webkit-tap-highlight-color: transparent;
```

Comment:
Why is the `-webkit` property required in this case?

Reply:
Because there are no tap-highlight-color in most browsers, we only want to target webkit ones for that property